scripts/datascience/convert_bin_datafile_to_json.py

```
@@ -0,0 +1,71 @@
+import gensim.downloader
```

Ruff (0.12.2)

`gensim.downloader` imported but unused

Remove unused import: `gensim.downloader`

(F401)